rules/manager_test.go

```diff
@@ -407,112 +407,118 @@ func TestForStateRestore(t *testing.T) {
 
```

Comment:
What is `exp` (few lines above) used for? I think it should be removed, because to my understanding expectation is checked against `testInput.alerts`.

Reply:
Good shout, I missed this.

Done in https://github.com/prometheus/prometheus/pull/13980/commits/f63dbc3db2abee643c7e182690be5712c42f13ca